src/tests/util.ts

```diff
@@ -38,5 +40,7 @@ export function whenState<State>(store: Store<State>, predicate: (state: State)
     reject('timeout')
   }, timeoutMillis)
 
+  callback()
```

Comment:
It's not clear why we need to call the callback

Reply:
The state may already pass the predicate function. So by calling the callback here, we may resolve the promise even if there are no subsequent state changes that trigger store.subscribe